spec/hashTable.js

```diff
@@ -11,19 +11,69 @@ describe('hashTable', () => {
 
   context('put()', () => {
     it('adds a key-value pair to the hash table.', () => {
-<<<<<<< Updated upstream
-      const myHash = new HashTable()
+      const ht = new HashTable()
+      ht.put('name', 'frankenstein')
+      expect(ht.size()).to.equal(1)
+      ht.put('19.0', 'stein')
+      expect(ht.size()).to.equal(2)
+      ht.put('wsqe', 'frank')
+      ht.put('tomp', 'more data')
+      ht.put('separate', 'hashes')
+      ht.put('difdeaaat', 'another one')
+      expect(ht.contains('tomp')).to.be.true
+      let stuff = ht.hashFunction('difdeaaat')
+    })
+  })
+
+  context('get()', () => {
+    it('returns the data associated with key.', () => {
```

Comment:
Again, I think this test is doing too much - one `put`/`get` should be sufficient to validate the functionality.  You should also be testing, in a separate test, the behavior of `get` in the event that the element is not found in the hash table.